app/mailers/user_notifications_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UserNotificationsMailer < ApplicationMailer
+  def submitted_event(event, user)
```

Comment:
The name "submitted_event" is a bit misleading. It could also be a notification for users that have submitted an event. we could call it 'new_event' or to be more specific 'new_event_in_your_country' (maybe that's an overkill?) or better 'new_local_event'.

Reply:
I will use 'new_local_event' 👍